webapp/src/main/webapp/src/app/groups/import/import-table/import-table.component.html

```diff
@@ -0,0 +1,39 @@
+<p-dataTable
+  #dt
+  [rows]="200"
+  [paginator]="imports.length > 200"
```

Comment:
Show the paginator only if we have > 200 rows.  Anything more can cause slowness as the datatable renders.